src/pages/Defi/components/EligibleCarousel.tsx

```diff
@@ -0,0 +1,49 @@
+import { Button } from '@chakra-ui/react'
+import { bnOrZero } from '@shapeshiftoss/investor-foxy'
+import { useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { NavLink } from 'react-router-dom'
+import { Card } from 'components/Card/Card'
+import { Carousel } from 'components/Carousel/Carousel'
+import { selectAggregatedEarnUserStakingEligibleOpportunities } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+import { FeaturedCard } from './FeaturedCards/FeaturedCard'
+
+export const EligibleCarousel = () => {
+  const translate = useTranslate()
+  const stakingOpportunities = useAppSelector(selectAggregatedEarnUserStakingEligibleOpportunities)
+  const filteredOpportunities = useMemo(() => {
+    return stakingOpportunities
+      .filter(o => bnOrZero(o.tvl).gt(50000))
+      .sort((a, b) => Number(b.apy) - Number(a.apy))
```

Comment:
`apy` is currently a float, use `bn()` instead of the Number cast, JS numbers are turbo unsafu